wGetStarted/L-Create-RetainFeature.md

```diff
@@ -104,19 +106,33 @@ To fix this, use the RetainedRowColumns to retain the rows and your formulas.
 ![](/images/L-Create-Retain/09.png)
 <br>
 
-**Step 2:** Now, click on the cell **G16,** which contains the ReportVariable() formula. Now click the **fx** button to pull up the function wizard.
+**Step 2:** Next we need to identify the key column(s) that will be used to signify which rows to be retained. Interject will retain any row with a value in any of these columns. Typically you would enter the name or list of names of the key columns. However, in this financial report, the key columns are using the [jColumnDef()](/wFunctions/jColumnDef.html) function. This function creates a composite column key made up of multiple values.
+
+You can still reference jColumnDef columns by using the names "jColumnDef_1", "jColumnDef_2", "jColumnDef_3", etc. To insert these names, you will create a new row that contains these names so you can reference them in the ReportVariable function.
+
+First right click row 3 and click **Insert** to insert a new row. 
+
+![](/images/L-Create-Retain/InsertRow.png)
```

Comment:
this is a small note but its a good practice to fill out the alt text for images incase a user's internet is slow to load images

ie: `![image of insert row](....)`

Reply:
I can add this to a task to update all the alt text as there isn't any throughout the entire doc site

Reply:
ya its not a high priority but its not a bad thing to have a task for... search all and replace might make this easier tho